### ISSUE TITLE
how to get the correspond image_id and label when using predict_generator and flow_from_directory

### ISSUE THREAD

**Issue by jamesben6688** [fchollet/keras#8352]
I am using keras.preprocess.image.ImageDataGenerator.flow_from_directory to generate image batches. However, when I try to use model.predict_generator to predict the labels, I do not know the label and the correspond image_id. Anyone help me? My code is as follows:

    from keras.preprocess.image import ImageDataGenerator

    from keras.applications.vgg19 import VGG19
    base_model = VGG19(...)
    data_gen = ImageDataGenerator(...)

    result = model.predict_generator(date_gen.flow_from_directory(valid_path), shuffle=False, target_size=
    (input_size, input_size))

However, I cannot get the image_id and the correspond probability? Anyone help me?
